learn/getting_started/quick_start.md

```diff
@@ -269,6 +269,24 @@ If you have multiple indexes, you can switch between them using the indexes drop
 
 ![Meilisearch's search preview indicating the indexes dropdown in the upper right corner](/search_preview/multiple_indexes.png)
 
+## Customization
+
+At this point, you can configure your Meilisearch instance, and customize your index and search responses.  
+
+### Instance options
+
+These allow you to configure your Meilisearch instance at launch with [environment variables](/learn/configuration/instance_options.md#environment-variables) and [command-line options](/learn/configuration/instance_options.md#command-line-options-and-flags). You need to relaunch your instance to alter them.
+
+Instance options **affect your entire Meilisearch instance**, not just a single index.
+
+### Index settings
+
+[Index settings](/reference/api/settings.md) allow you to customize search behavior. You can either update all settings globally using the [update settings endpoint](/reference/api/settings.md#update-settings) or individually using a specific child route.
+
+### Search parameters
+
+[Search parameters](/reference/api/search.md#search-parameters) are used with the search endpoints to improve relevancy. They allow you to alter search responses and behavior.
```

Comment:
I _think_ "search results" is more accurate than "search responses"—I'm afraid the second could be understood as the actual response instead of its contents.

```suggestion
[Search parameters](/reference/api/search.md#search-parameters) are used with the search endpoints to improve relevancy. They allow you to alter search results and behavior.
```